WalletWasabi.Fluent/Views/Wallets/Home/History/Details/TransactionDetailsView.axaml

```diff
@@ -79,6 +79,18 @@
 
       <Separator />
 
+      <!-- Confirmation Time -->
+      <c:PreviewItem IsVisible="{Binding ConfirmationTime, Converter={x:Static ObjectConverters.IsNotNull}}"
+                     Icon="{StaticResource clock_regular}"
```

Comment:
Use the same icon as on the transaction preview

Reply:
Fixed.